include/core/AABB.hpp

```diff
@@ -1,3 +1,33 @@
+/*************************************************************************/
+/*  AABB.hpp                                                             */
+/*************************************************************************/
+/*                       This file is part of:                           */
+/*                           GODOT ENGINE                                */
```

Comment:
Should we changed this to `GODOT-CPP` instead?

Reply:
I think so, or something along the lines of "Godot C++ Extension library"

Reply:
I think it's fine to keep the generic "GODOT ENGINE" as referring to the project.

I'm fine if we want to go with "Godot C++ Extension Library" as suggested though.